Lib/test/test_ssl.py

```diff
@@ -123,6 +124,24 @@ def asn1time(cert_time):
 
     return cert_time
 
+def bytebuf(filepath):
+    """
+    Get re-read()able buffer object, with bytes read from `filepath`.
+    """
+    with open(filepath, 'rb') as f:
+        buf = BytesIO(f.read())
+    buf.read = buf.getvalue
```

Comment:
I think I wanted to have the object be re-`read()`able, because IIRC that simplified writing the test code quite a bit:

```
>>> import io
>>> b = io.BytesIO(b"abc")
>>> b.read()
b'abc'
>>> b.read()
b''
```
vs.
```
>>> b = io.BytesIO(b"abc")
>>> b.read = b.getvalue
>>> b.read()
b'abc'
>>> b.read()
b'abc'
```